manifest.json

```diff
@@ -33,7 +33,8 @@
         }
     ],
     "background": {
-        "scripts": ["background.js"]
```

Comment:
Is the scripts shortcut not supported by Edge?

Reply:
Not sure, there converted replaced this I think.

Reply:
Can you do some research to see if "scripts" would work just as well?